Add missing python query `format_expression`

Danielkonge opened the pull request:
I found a missing python query, so this is just a minor fix.

HiPhish commented:
You have forked from quite an old state. Can you please fork off the current `master`? And while you're at it please update the sample file as well to contain a `format_expression`. I don't know what such a node looks like.

Danielkonge commented:
> You have forked from quite an old state. Can you please fork off the current `master`?

Ah, I accidentally pulled from my fork's master branch instead of upstream. It should be fixed now.

> And while you're at it please update the sample file as well to contain a `format_expression`. I don't know what such a node looks like.

It's been some time since I added a query, so I forgot about this. I have added it now.

HiPhish commented:
Oh nice, I did not know about that one. Thank you.
